flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,540 @@
+                        if (
+                            self.config.get('include_available')
+                            and parent_request.get('available')
+                            or not parent_request.get('available')
+                        ):
+                            entry = self.generate_movie_entry(parent_request)
+                            self._items.append(entry)
+                elif self.config.get('type') == 'shows':
+                    # Shows do not have approvals or available flags so include them all
+                    entry = self.generate_tv_entry(parent_request)
+                    self._items.append(entry)
+                else:
+                    for child_request in parent_request["childRequests"]:
+                        for season in child_request["seasonRequests"]:
+                            # Seasons do not have approvals or available flags so include them all
+                            if self.config.get('type') == 'seasons':
+                                entry = self.generate_tv_entry(
+                                    parent_request, child_request, season
+                                )
+                                if entry:
+                                    self._items.append(entry)
+                            else:
+                                for episode in season['episodes']:
+                                    # check that the request is approved unless user has selected to include everything
+                                    if (
+                                        self.config.get('only_approved')
+                                        and not episode.get('approved')
+                                        or episode.get('approved')
+                                    ):
+                                        # Always include items that are not available and only include available items if user has selected to do so
+                                        if (
+                                            self.config.get('include_available')
+                                            and episode.get('available')
+                                        ) or not episode.get('available'):
+                                            entry = self.generate_tv_entry(
+                                                parent_request, child_request, season, episode
+                                            )
+                                            self._items.append(entry)
+        return self._items
+
+    @property
+    def online(self):
+        """Set the online status of the plugin, online plugin should be treated differently in certain situations,
+        like test mode"""
+        return True
+
+    # -- Public interface ends here -- #
+
+    def _get_ombi_entry(self, entry: Entry) -> Optional[OmbiEntry]:
+
+        if "tmdb_id" not in entry:
+            log.warning(
```

Comment:
You might want to use log_once to avoid spamming this into log

Reply:
I'm pretty sure if you are trying to mark 5 items as watched and 3 are missing `tmbd_id` then you will want to see the log three times.

But your comment did help me see I'm missing a return on this branch 👍